package.json

```diff
@@ -4,7 +4,7 @@
   "description": "ReasonML web framework",
   "license": "MIT",
   "esy": {
-    "build": "dune build --only-packages #{self.name}",
+    "build": "dune build -p #{self.name}",
```

Comment:
I don’t think this is acceptable, because it makes development slower. Running `esy` at the root of the project will now perform a release build.

Reply:
The solution is to write a proper opam file and have people depend on the opam file

Reply:
Another reason is that it doesn't really make sense to do a release build of ReWeb. We will normally consume it as a library dep and do a release build of the consuming app, e.g. fullstack-reason.

Reply:
Esy supports `buildDev` for this purpose

Reply:
What about something like here? https://github.com/esy-ocaml/hello-reason/pull/61/files

Reply:
That looks great. So just to clarify, after this:

- `esy build` will run `dune build --root . --only-package #{self.name}`
- `esy build --release` will run `dune build -p #{self.name}`

Reply:
I've added `buildDev` and according to https://esy.sh/docs/en/configuration.html#esybuilddev this should indeed fix the issue.